homeassistant/components/flux_led/manifest.json

```diff
@@ -1,8 +1,9 @@
 {
   "domain": "flux_led",
-  "name": "Flux LED/MagicLight",
+  "name": "Flux LED/MagicHome",
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/flux_led",
   "requirements": ["flux_led==0.22"],
-  "codeowners": [],
+  "codeowners": ["@djtimca", "@icemanch"],
```

Comment:
Is djtimca involved with this PR?

Reply:
Most of the code in this pull request is belongs to him and I pulled it from a different Pull Request that had gone stale.  Which probably went stale because of me trying to update the Flux_LED API.

Reply:
We define code ownership like this:
> The willingness of a contributor to try, at best effort, to maintain the integration. Providing the intention for handling issues, providing bug fixes, or other contributions to the integration one is listed on as a code owner.

Unless @djtimca has agreed to this responsibility, let's not add him now.